src/Module.php

```diff
@@ -108,4 +118,9 @@ public function onAuthenticationPost(MvcAuthEvent $e)
 
         $this->container->setService('api-identity', $e->getIdentity());
     }
+
+    public function getMvcRouteListener()
```

Comment:
There is; we already had the `$container` property, and can do an `assertAttributeEmpty()` on that.